PKGBUILD

```diff
@@ -1,5 +1,5 @@
 pkgname=modular
-pkgver=0.4.1
+pkgver=0.7.0
```

Comment:
I think, you meant to update the Mojo version.

The newest modular version I have seen was 0.5.0, but they didn't announced it in any way (I think)

Reply:
Yea, at first I wasn't sure, then I went back and changed it to 0.4.1 when I found that it was the modular version. I also see 0.5.0 as well in [this announcement](https://github.com/modularml/mojo/discussions/1214).